samples/snippets/csharp/language-reference/tokens/string-interpolation.cs

```diff
@@ -61,13 +61,13 @@ private static void DelimitInterpolatedExpression()
     private static void CreateCultureSpecificResults()
     {
         // <Snippet4>
-        double price = 1000;
-        FormattableString message = $"The cost of this item is {price:C}.";
+        double speedOfLight = 299792.458;
```

Comment:
I considered speed of light, but not speed of light in km/s. It's definitely a better choice than the constant I suggested.